source/frontend/src/features/properties/map/acquisition/update/stakeholders/InterestHolderPropertiesTable.tsx

```diff
@@ -0,0 +1,51 @@
+import OverflowTip from 'components/common/OverflowTip';
+import { ColumnWithProps, Table } from 'components/Table';
+import { Api_Property } from 'models/api/Property';
+import { Api_PropertyFile } from 'models/api/PropertyFile';
+import * as React from 'react';
+import { CellProps } from 'react-table';
+import { getFilePropertyName } from 'utils/mapPropertyUtils';
+
+interface IInterestHolderPropertiesProps {
+  fileProperties: Api_PropertyFile[];
+  setSelectedFileProperties: (properties: Api_PropertyFile[]) => void;
+  selectedFileProperties: Api_PropertyFile[];
+  disabledSelection: boolean;
+}
+
+const columns: ColumnWithProps<Api_PropertyFile>[] = [
+  {
+    Header: 'Identifier',
+    accessor: 'property',
+    align: 'left',
+    minWidth: 60,
+    maxWidth: 60,
+    Cell: (cellProps: CellProps<Api_PropertyFile, Api_Property | undefined>) => {
+      const propertyName = getFilePropertyName(cellProps.row.original);
+      return <OverflowTip fullText={`${propertyName.label}: ${propertyName.value}`} />;
+    },
```

Comment:
question - did you test properties with "friendly" names; eg "Corner lot of land"?
This code seems to indicate that those properties would display as
```
[checkbox] "Descriptive Name: Corner Lot"
```

instead of 
```
[checkbox] "Corner Lot"
```

which is how it shows in UXpin

Reply:
I'll check in with Bobbi on this one - was hoping for consistency with our other logic.